staging/src/k8s.io/client-go/rest/client.go

```diff
@@ -239,3 +240,28 @@ func (c *RESTClient) Delete() *Request {
 func (c *RESTClient) APIVersion() schema.GroupVersion {
 	return c.content.GroupVersion
 }
+
+// todo doc, name
+type clientNegotiatorWithCBORSequenceAlias struct {
```

Comment:
we're going to have this forever?

Reply:
We will need something to allow clients to choose the correct stream decoder if they see this response content type. There are other approaches. It could be a new field on StreamSerializerInfo that specifies a distinct stream media type, but I did not do that because none of the other data formats would use it and it would be more invasive.